### PR TITLE
Issue #13213: Remove '//ok' comments from Input files (InputGenericWhitespace)

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Part of https://github.com/checkstyle/checkstyle/issues/13213

Link to check documentation: https://checkstyle.org/checks/whitespace/genericwhitespace.html

```
➜  checkstyle git:(master) grep genericwhitespace config/checkstyle-input-suppressions.xml
            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceEndsTheLine.java"/>
            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceInnerClass.java"/>
            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceList.java"/>
            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceMethodRef1.java"/>
            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceWithEmoji.java"/>
➜  checkstyle git:(master) echo $?                                                             
0

➜  checkstyle git:(remove-ok-InputGenericWhitespaceList) grep genericwhitespace config/checkstyle-input-suppressions.xml
➜  checkstyle git:(remove-ok-InputGenericWhitespaceList) echo $?      
1
```

### CODE REVIEW

**Comment by romani** [comment]
Always base your PR on master not your previous PRs.
All changes are independent and can be merged separately.

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -7841,16 +7841,6 @@
             files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]packageinfo[\\/]test1[\\/]package-info.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]packageinfo[\\/]test2[\\/]package-info.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (nrmancuso) [inline config/checkstyle-input-suppressions.xml:7844]: Please show `grep` or some other command line search utility for `genericwhitespace` in this file.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (relentless-pursuit) [reply config/checkstyle-input-suppressions.xml:7844]: done.

Comment (nrmancuso) [reply config/checkstyle-input-suppressions.xml:7844]: Use https://github.com/checkstyle/checkstyle/pull/13609#issue-1866293244 as an example, always show "before" and "after", I think you will find a surprise :)

Comment (relentless-pursuit) [reply config/checkstyle-input-suppressions.xml:7844]: done

File: config/checkstyle-input-suppressions.xml [hunk 2]
```diff
@@ -7840,17 +7840,7 @@
...
-            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceInnerClass.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceList.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceMethodRef1.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]genericwhitespace[\\/]InputGenericWhitespaceWithEmoji.java"/>
+            files="checks[\\/]whitespace[\\/]emptylineseparator[\\/]packageinfo[\\/]test2[\\/]package-info.java"/>q
   <suppress id="UnnecessaryOkComment"
```
Comment (nrmancuso) [inline config/checkstyle-input-suppressions.xml:7844]: `mvn clean verify`  should always be passing on local before pushing changes.

**Review by nrmancuso — CHANGES_REQUESTED** [review]

**Review by nrmancuso — APPROVED** [review]
Ok to merge as CI passes

**Comment by relentless-pursuit** [comment]
> Ok to merge as CI passes

should i now procced pacakge wise or class wise? i have seen comments which said too big a PR. so, what can a right approach?

**Comment by nrmancuso** [comment]
@relentless-pursuit I would stick to check level in any case, smaller PRs get merged faster.

**Review by romani — APPROVED** [review]

**Comment by romani** [comment]
Please do few more by Check, as you get to level to do update without any comment, we can do by package or by few Checks.

**Pull request merged**